.cloud-build/cleanup/cleanup.py

```diff
@@ -42,7 +42,7 @@ def run_cleanup_managers(managers: List[ResourceCleanupManager], is_dry_run: boo
         print("")
 
 
-is_dry_run = False
+is_dry_run = Falsee
```

Comment:
Falsee?

Reply:
Can you run a test before you merge?

We really need PR tests for the cleanup code.